src/T2DMap.cpp

```diff
@@ -791,7 +791,7 @@ void T2DMap::paintEvent(QPaintEvent* e)
                 } else if (mCustomLineSelectedExit == "OUT" || mCustomLineSelectedExit == "out") {
                     customLineDestinationTarget = pSR->getOut();
                 } else {
-                    QMultiMap<int, QString> otherExits = pSR->getOtherMap();
+                    const QMultiMap<int, QString> &otherExits = pSR->getOtherMap();
                     QMapIterator<int, QString> otherExitIt = otherExits;
```

Comment:
Actually, is `otherExits` needed - I now know that I could have used the (reference?) item directly as:

    QMapIterator<int, QString> otherExitIt(pSR->getOtherMap());

here and below...

Reply:
It's not, yeah. I didn't want to mess with it too much :)

Fixed